docs/reference/auth/index.rst

```diff
@@ -256,23 +256,33 @@ You can configure webhooks with the UI or via query.
 Configuring SMTP
 ================
 
-For email-based factors, you can configure SMTP to allow the extension to send
-emails on your behalf. You should either configure SMTP, or webhooks for the
-relevant events.
 
-Here is an example of configuring SMTP for local development, using something
-like `Mailpit <https://mailpit.axllent.org/docs/>`__.
+For email-based factors, you can configure SMTP to allow the extension to send emails on your behalf. You should either configure SMTP, or webhooks for the relevant events.
+
+The easiest way to configure SMTP is to use the built-in UI. Here is an example of configuring SMTP for local development using EdgeQL directly, using something like `Mailpit <https://mailpit.axllent.org/docs/>`__.
```

Comment:
Do we have SMTP auto-configured in our cloud? if yes (iirc it's "yes") this is a perfect opportunity to pitch our cloud with a "note" section.

Reply:
Yeah good call! We should also add a section about SMTP in the Cloud docs, and maybe we can lay out the specifics of the restrictions on the dev-only SMTP proxy we provide there?

Reply:
yes to all of that

Reply:
@1st1 ⚡ [`d42c0e6` (#8535)](https://github.com/geldata/gel/pull/8535/commits/d42c0e6176961df3db51daabfa1be873e95c4849)

<img width="847" alt="image" src="https://github.com/user-attachments/assets/de50e316-82d5-4fd3-91a7-af8843fbcee7" />

I'll follow up with some Cloud docs about the SMTP provider, but this seems like a fine start. wdyt?